### PR TITLE
Bug 1438862-Refresh button doesn't seem to be working in the client manager

### PR DESCRIPTION
**Pull request by Biboswan**

### CODE REVIEW

**Comment by Biboswan** [comment]
@helfi92  previously `this` was becoming null inside loadClients .So refresh button wasn't loading clients again. Now its done as you said i think.

File: src/views/ClientManager/ClientManager.js [hunk 1]
```diff
@@ -67,23 +68,24 @@ export default class ClientManager extends Component {
   }
 
   async loadClients() {
```
Comment (helfi92) [inline src/views/ClientManager/ClientManager.js]: Could you change this line to:

```js
loadClients = async () => {
```

Using arrow functions  has the benefit of not changing the meaning of `this`, so things just work automatically.

Comment (helfi92) [reply src/views/ClientManager/ClientManager.js]: You will then be able to delete  `this.loadClients = this.loadClients.bind(this);`.

File: src/views/ClientManager/ClientManager.js [hunk 2]
```diff
@@ -66,25 +66,26 @@ export default class ClientManager extends Component {
...
-        error: null
-      });
-    } catch (err) {
-      this.setState({
-        clients: null,
-        error: err
-      });
-    }
-  }
+  loadClients = async () => {
```
Comment (eliperelman) [inline src/views/ClientManager/ClientManager.js]: This method isn't doing anything async, so we can remove that from the function definition.

```js
loadClients = () => {
```

Comment (helfi92) [reply src/views/ClientManager/ClientManager.js]: Agreed. @Biboswan let's remove it :)

**Pull request merged**